build-tools/create-pkg/create-pkg.targets

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\installers\create-installers.targets" />
+  <PropertyGroup>
+    <BuildDependsOn>ResolveReferences</BuildDependsOn>
+  </PropertyGroup>
+  <PropertyGroup>
+    <PkgNamePrefix Condition="'$(PkgNamePrefix)' == ''">xamarin.android-oss</PkgNamePrefix>
+    <PkgInstallDir>/</PkgInstallDir>
+    <PkgInstallDir Condition="$([MSBuild]::IsOsPlatform(Linux))">/usr</PkgInstallDir>
+    <PayloadDir>$(OutputPath)/pkg/archive</PayloadDir>
+    <PkgOutputPath>$(OutputPath)/pkg/packages</PkgOutputPath>
+    <PkgResourcesPath>$(OutputPath)/pkg/resources</PkgResourcesPath>
+    <PkgDistributionDestination>$(OutputPath)/pkg/distribution.xml</PkgDistributionDestination>
+    <PkgLicenseSrcEn Condition="'$(PkgLicenseSrcEn)' == ''">$(XamarinAndroidSourcePath)\LICENSE</PkgLicenseSrcEn>
+    <PkgLicenseDestinationEn>$(PkgResourcesPath)/en.lproj</PkgLicenseDestinationEn>
+    <UpdateInfoGuid>d1ec039f-f3db-468b-a508-896d7c382999</UpdateInfoGuid>
+  </PropertyGroup>
+  <Target Name="_CopyFilesToPayloadDir"
+      DependsOnTargets="GetXAVersion" >
+    <PropertyGroup>
+      <XAFrameworkDir>$(PayloadDir)/Library/Frameworks/Xamarin.Android.framework/Versions/$(XAVersion)</XAFrameworkDir>
```

Comment:
For consistency, we should use `\` for directory separator chars ~everywhere within MSBuild projects.

Reply:
I've made these changes everywhere outside of Exec calls which are only going to run on Unix.